_posts/2020/11/2020-11-12-hpccarpentry-lesson-development.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title:  "Applying lesson development design principles"
+authors: ["Alan O'Cais", "Peter Steinbach"]
+date: 2020-11-12
+time: "00:10:00" 
+teaser: "Thinking about the practical application of Bloom's Taxonomy"
+tags: ["Community", "Lesson development"]
+---
+
+Together with CodeRefinery ([1]), we helped to organise
```

Comment:
For these relative links, I believe you need both the link-text *and* the link alias. So, a preferable format for these would be `([1][1])` instead of `([1])`.

Reply:
Ooh, nevermind! I learned something today!